### PR TITLE
doc: `CryptoFromPool.vy`

### PR DESCRIPTION
**Pull request by mo-anon**
resolves https://github.com/CurveDocs/curve-docs/issues/150

### CODE REVIEW

File: docs/lending/contracts/oracle.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
+<h1>Curve Lending: Oracles</h1>
+
+!!!github "GitHub"
+    The source code of the `CryptoFromPool.vy` and all other oracle contracts can be found on [GitHub :material-github:](https://github.com/curvefi/curve-stablecoin/blob/lending/contracts/price_oracles/).
+
+The [`OneWayLendingFactory.vy`](./oneway-factory.md) has a [`create_from_pool`](./oneway-factory.md#create_from_pool) method which deploys the full lending market infrastucture alsong with a price oracle using a `stableswap-ng`, `twocrypto-ng` or `tricrypto-ng` pool. These pools all have a suitable exponential moving-average (EMA) oracle, which can be used in lending markets.
```
Comment (saint-rat) [inline docs/lending/contracts/oracle.md:6]: >alsong

File: docs/lending/contracts/oracle.md [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+        ```shell
+        >>> CryptoFromPool.COLLATERAL_IX()
+        2
+        ```
+
+
+### `N_COINS`
+!!! description "`CryptoFromPool.N_COINS() -> uint256: view`"
+
+    Getter for the total number of coin in the liquidity pool.
```
Comment (saint-rat) [inline docs/lending/contracts/oracle.md:335]: > total number of coin

total number of coins

File: docs/lending/contracts/oracle.md [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+        ```shell
+        >>> CryptoFromPool.N_COINS()
+        3
+        ```
+
+
+### `NO_ARGUMENT`
+!!! description "`CryptoFromPool.NO_ARGUMENT() -> bool: view`"
+
+    Getter for the info on <todo>. The variable is set to `false` if the pool where the price oracle is taken from only has two coins.
```
Comment (saint-rat) [inline docs/lending/contracts/oracle.md:384]: >info on <todo>
not sure what's going on here

Comment (mo-anon) [reply docs/lending/contracts/oracle.md:384]: lmao yeah no clue either

**Pull request merged**